languages/es-ES.js

```diff
@@ -1,169 +1,169 @@
+			].join('\n'),
+			COMMAND_EVAL_ERROR: (time, output, type) => `**Error**:${output}\n**Tipo**:${type}\n${time}`,
+			COMMAND_EVAL_OUTPUT: (time, output, type) => `**Salida**:${output}\n**Tipo**:${type}\n${time}`,
+			COMMAND_EVAL_SENDFILE: (time, type) => `La salida era demasiado larga... enviada como archivo.\n**Tipo**:${type}\n${time}`,
+			COMMAND_EVAL_SENDCONSOLE: (time, type) => `La salida era demasiado larga... imprimiendo el resultado en consola.\n**Tipo**:${type}\n${time}`,
+			COMMAD_UNLOAD: (type, name) => `✅ Desactivada la pieza tipo ${type}: ${name}`,
+			COMMAND_UNLOAD_DESCRIPTION: 'Desactiva una pieza de Klasa.',
+			COMMAND_UNLOAD: (type, name) => `✅ Desactivada la pieza tipo ${type}: ${name}`,
+			COMMAND_TRANSFER_ERROR: '❌ El archivo ha sido transferido o nunca existió.',
+			COMMAND_TRANSFER_SUCCESS: (type, name) => `✅ Transferida la pieza tipo ${type}: ${name} con éxito.`,
+			COMMAND_TRANSFER_FAILED: (type, name) => `La transferencia de la pieza tipo ${type}: ${name} al Cliente ha fallado. Por favor, revisa la consola.`,
+			COMMAND_RELOAD: (type, name) => `✅ Recargada la pieza tipo ${type}: ${name}`,
+			COMMAND_RELOAD_ALL: (type) => `✅ Recargadas todas las piezas tipo ${type}.`,
+			COMMAND_RELOAD_DESCRIPTION: 'Recarga una pieza de Klasa, o todas las piezas de una colección.',
+			COMMAND_REBOOT: 'Reiniciando...',
+			COMMAND_REBOOT_DESCRIPTION: 'Reinicia el bot.',
+			COMMAND_PING: '¿Ping?',
+			COMMAND_PING_DESCRIPTION: 'Ejecuta una prueba de conexión a Discord.',
+			COMMAND_PINGPONG: (diff, ping) => `¡Pong! (El viaje duró: ${diff}ms. Latencia: ${ping}ms.)`,
+			COMMAND_INVITE_SELFBOT: '¿Para qué quiere un enlace de invitación un selfbot?',
+			COMMAND_INVITE: (client) => [
+				`Para añadir ${client.user.username} a tu servidor de Discord:`,
+				`<${client.invite}>`,
+				util.codeBlock('', [
+					'El enlace de arriba está generado con los permisos necesarios para ejecutar todos los comandos actuales.',
+					'Entiendo que no todos los permisos son necesarios para todos los servidores, pero puedes cambiarlos sin problema.',
+					'Si intentas usar un comando que requira más permisos de los que tengo, te lo haré saber.'
+				].join(' ')),
+				'Por favor reporta a <https://github.com/dirigeants/klasa> si encuentras algún bug.'
+			],
+			COMMAND_INVITE_DESCRIPTION: 'Muestra el enlace de invitación para el bot.',
+			COMMAND_INFO: [
+				'Klasa es un framework \'plug-and-play\' construido encima de la librería Discord.js.',
+				'La mayor parte del código es modularizado, lo cual permite a los desarrolladores editar Klasa para encajar con sus necesidades.',
+				'',
+				'Algunas de las características de Klasa incluyen:',
+				'• 🐇💨 Carga muy rápida con soporte ES2017 (`async`/`await`).',
+				'• 🎚🎛 Configuración separada para cada servidor, usuario y cliente que puede ser expandida fácilmente.',

```

Comment:
I think we use the Oxford comma in Spanish, could be wrong.

```suggestion
				'• 🎚🎛 Configuración separada para cada servidor, usuario, y cliente que puede ser expandida fácilmente.',
```

If we do, this change should reflect to the lines below too.

Reply:
This, that and no comma! 🤣

Reply:
Por favor, in English the Oxford comma is required, but apparently it's not in Spanish, unless you want to give emphasis.

I guess it's just my journey learning English what made this lack weird for me, but since this is allowed in Spanish, I'll let it go 😅